popsycle/synthetic.py

```diff
@@ -1085,6 +1155,9 @@ def _make_comp_dict(iso_dir, log_age, currentClusterMass,
         The next unique ID number (int) that will be assigned to
         the new compact objects created.
 
+    unmade_cluster_counter: int
```

Comment:
Can you also calculate the total mass in the unmade clusters, in addition to the number?